docker/Dockerfile.centos6

```diff
@@ -20,6 +20,11 @@ RUN yum install -y \
  wget \
  zip
 
+RUN mkdir -p /var/cache/yum/x86_64/6/centos-sclo-rh/ && \
```

Comment:
I ended up adding the `devtools-7` package here too in order to use a newer `gcc` version for the GraalVM gate. I think this should only affect commands executed by `/usr/bin/scl enable devtoolset-7 <command>`
Would something like this be okay or should we use a different approach here?

Reply:
I think that's fine.